exporter/importables.go

```diff
@@ -1480,4 +1491,52 @@ var resourcesMap map[string]importable = map[string]importable{
 			{Path: "library.whl", Resource: "databricks_dbfs_file", Match: "dbfs_path"},
 		},
 	},
+	"databricks_directory": {
+		Service: "directories",
+		Name: func(ic *importContext, d *schema.ResourceData) string {
+			name := d.Get("path").(string)
+			if name == "" {
+				return d.Id()
+			} else {
+				name = nameNormalizationRegex.ReplaceAllString(name[1:], "_") + "_" +
+					strconv.FormatInt(int64(d.Get("object_id").(int)), 10)
+			}
+			return name
+		},
+		List: func(ic *importContext) error {
+			notebooksAPI := workspace.NewNotebooksAPI(ic.Context, ic.Client)
+			directoryList, err := notebooksAPI.ListDirectories("/", true)
+			if err != nil {
+				return err
+			}
+			for offset, directory := range directoryList {
+				if strings.HasPrefix(directory.Path, "/Repos") {
+					continue
+				}
+				ic.Emit(&resource{
+					Resource: "databricks_directory",
+					ID:       directory.Path,
+				})
+				if offset%50 == 0 {
+					log.Printf("[INFO] Scanned %d of %d directories",
+						offset+1, len(directoryList))
+				}
+			}
+			return nil
+		},
+		Import: func(ic *importContext, r *resource) error {
```

Comment:
Import won't be called if you don't do emit on directory (not listing)

Reply:
what's the issue here?